src/Ninja/G1CP/Content/Testsuite/testsuite.d

```diff
@@ -275,3 +284,18 @@ func string G1CP_TestsuiteList(var string _) {
 
     return ret;
 };
+
+func void G1CP_Testsuite_CheckPassed() {
```

Comment:
We should add a header comment here with a brief description of the function (see other functions of this file).